authN-authZ/auth-istio/README.md

```diff
@@ -12,23 +12,46 @@ Here we use the chatQnA pipeline as an example.
 
 ## Prerequisite
 
-Before composing an OPEA pipeline with authN & authZ, user need to install Istio to support this feature. Please follow the steps [here](https://istio.io/latest/docs/setup/install/istioctl/) for Istio installation.
+**Istio installation**
+
+Before composing an OPEA pipeline with authN & authZ, user need to install Istio to support this feature.
+
+```bash
+curl -L https://istio.io/downloadIstio | sh -
+cd istio-1.23.2
+export PATH=$PWD/bin:$PATH
+istioctl install
+```
+
+You can refer to [Istio installation](https://istio.io/latest/docs/setup/install/istioctl/) for detail information.
 
 **Deploy chatQnA pipeline and enable Istio sidecar injection**
 
-```sh
+Create a new namespace and you can enable istio sidecar injection for the namespace directly.
 
-# deploy ChatQnA pipeline. You can either leverage GMC or the ChatQnA helm chart.
+```sh
 kubectl create ns chatqa
-# here's the command to leverage GMC custom resource for ChatQnA deployment.
+kubectl label namespace chatqa istio-injection=enabled
```

Comment:
This would enable istio injection for all pods within that namespace. Any reason for that?

Reply:
This is just for convenience, I have just add "optional" to this part.